internal/lvmd/command/lvm_vgs.go

```diff
@@ -53,7 +53,7 @@ func getVGReport(ctx context.Context, name string) (vg, error) {
 	args := []string{
 		"vgs", name, "-o", "vg_uuid,vg_name,vg_size,vg_free", "--units", "b", "--nosuffix", "--reportformat", "json",
 	}
-	err := callLVMInto(ctx, res, args...)
+	err := callLVMInto(ctx, res, verbosityLVMStateUpdate, args...)
```

Comment:
This should be `verbosityLVMStateNoUpdate`, because `vgs` won't change the LVM's state.

Reply:
done

Reply:
done